colobot-base/src/script/scriptfunc.cpp

```diff
@@ -3527,6 +3570,10 @@ void CScriptFunctions::Init()
     CBotProgram::DefineNum("ResearchBuilder",       RESEARCH_BUILDER);
     CBotProgram::DefineNum("ResearchTarget",        RESEARCH_TARGET);
 
+    CBotProgram::DefineNum("DefaultView",           DefaultView);
+    CBotProgram::DefineNum("FirstPersonView",       FirstPersonView);
+    CBotProgram::DefineNum("ThirdPersonView",       ThirdPersonView);
```

Comment:
CBot constants that act as enumerators should preferably have a common prefix rather than a suffix. And I'd use `Camera` rather than `View` for the prefix since this changes the camera type and the function that uses it has `camera` in the name. So something like:

`DefaultView` -> `CameraDefault`
`FirstPersonView` -> `CameraFirstPerson`
`ThirdPersonView` -> `CameraThirdPerson`

Reply:
Agreed. Tho I'd suggest naming the first and third person constants `CameraOnboard` and `CameraBack` respectively as these are the terms used both in Scene and behind the scenes (heh).

Reply:
Done